packages/themeables/core/meta.tsx

```diff
@@ -39,34 +41,45 @@ const newTheme: TThemeables<TThemeableBackground, Mappings> = {
   }),
 }
 
-type TDemoComponent = TDemo & { hasTheme: boolean}
+type TDemoComponent = TDemo & { hasTheme: boolean, withOverride: boolean}
 
-export const Component = ({ status, hasTheme }: TDemoComponent) => (
-  <Block
-    style={{
-      width: 100,
-      height: 100,
-    }}
-  >
-    {(
+const overrideTheme: TOverrideables<TThemeableBackground, Mappings> = {
+  [SYMBOL_DEMO]: ({ status }) => ({
+    color: status === 'error' ? [144, 24, 56, 1] : [108, 105, 86, 1],
+  }),
+}
+
+export const Component = ({ status, hasTheme, withOverride }: TDemoComponent) => {
+  return (
+    <OverrideContext.Provider value={withOverride ? overrideTheme : {}}>
```

Comment:
shouldnt that one be on line 62 and not wrapping it all?

Reply:
I'm not sure why this matters, there is no right/wrong here. Only requirement is that any themeable is wrapped with it. 

But for the sake of removing any confusion i can move it

Reply:
I just think it might make it a bit clearer, thanks for understanding